### DISCUSSION TITLE
Infill & inner wall & outer wall fan speed

### DISCUSSION THREAD

**Discussion by john3dc** [OrcaSlicer/OrcaSlicer#5163] [Ideas]
hello. i would be very pleased to get a regulation for fan speeds for inner and outer walls like in superslicer?

![Unbenannt](https://github.com/SoftFever/OrcaSlicer/assets/142935668/41cb83fa-cf62-48d7-8b7e-7ff62c835c33)

**Comment by Dib-Roy** [comment]
I would also appreciate this feature because it lets you have a uniform outer surface with better layer adhesion on the interior.

**Comment by Avpman2** [comment]
+1

**Comment by Keepsv** [comment]
+1
